src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala

```diff
@@ -29,7 +29,7 @@ object SbtHeader extends AutoPlugin {
   object autoImport {
 
     object HeaderPattern {
-      val cStyleBlockComment = """(?s)(/\*(?!\*).*?\*/(?:\n|\r|\r\n)+)(.*)""".r
```

Comment:
I believe this defeats the purpose of the negative lookahead of a second "*" ;-)
So, I think your PR will lead to deleting JavaDoc/ScalaDoc comments and replacing these with the header.